DOC-2706 added `ZK.SnapCount` in config parameters (#836)
tigergraph/server-docs: modules/reference/pages/configuration-parameters.adoc

|ZK.QuorumPort |The port for Zookeeper to do peer communication |`2888`

|ZK.StartTimeoutMS |Start zookeeper timeout |`120000`

> |ZK.SnapCount |Defines the number of committed transactions that must occur before a snapshot is triggered and saved to disk. Once this threshold is reached, ZooKeeper creates a snapshot and starts a new transaction log file.

Reviewer: ```suggestion
|ZK.SnapCount |The number of committed transactions written to the log file that triggers a snapshot to be saved to disk. After the snapshot is saved, ZooKeeper starts a new transaction log file.
```
